PWGCF/GenericFramework/Core/FlowPtContainer.cxx

```diff
@@ -100,7 +116,12 @@ void FlowPtContainer::Initialise(const o2::framework::AxisSpec axis, const int&
     for (auto m(1); m <= mpar; ++m) {
       if (!(configs.GetpTCorrMasks()[i] & (1 << (m - 1))))
         continue;
-      fCovList->Add(new BootstrapProfile(Form("%s_mpt%i", configs.GetHeads()[i].c_str(), m), Form("%s_mpt%i", configs.GetHeads()[i].c_str(), m), nMultiBins, &multiBins[0]));
+      if (fUseCentralMoments) {
+        for (auto j = 0; j < m; ++j)
+          fCovList->Add(new BootstrapProfile(Form("%spt%i_Mpt%i", configs.GetHeads()[i].c_str(), m, m - j - 1), Form("%spt%i_Mpt%i", configs.GetHeads()[i].c_str(), m, m - j - 1), nMultiBins, &multiBins[0]));
```

Comment:
For the next iteration please, put curl brackets around this sentence. They improve readability